### PR TITLE
doc:tutorial: sentiment analysis on movie reviews

### PR DESCRIPTION
**Pull request by akshay-bharadva**
### A tutorial about Sentiment Analyses on Movie Reviews using OpenAI and MongoDB as a datasource.

**Fixes** #4940 

## Type of change

(Please delete options that are not relevant)

- [x] 📄 This change requires a documentation update

### What is the solution?

(Describe at a high level how the feature was implemented)

## Checklist:

- [ ] My code follows the style guidelines(PEP 8) of MindsDB.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [x] I have updated the documentation, or created issues to update them.
- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ] I have shared a short loom video or screenshots demonstrating any new functionality.

### CODE REVIEW

File: docs/tutorials/sentiment-analysis-of-movie-reviews.mdx [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+
+You can use [Mongo Compass](/connect/mongo-compass) or [Mongo Shell](/connect/mongo-shell) to connect your database, which we downloaded from [`Hugging Face`](https://huggingface.co/datasets/rotten_tomatoes), like this:
+
+## Connecting a Database
+
+We start by connecting a demo database using the CREATE DATABASE statement:
+
+- `host` : connection string for the mongodb
+
+```bash
```
Comment (martyna-mindsdb) [inline docs/tutorials/sentiment-analysis-of-movie-reviews.mdx:28]: Please note that it is a tutorial using MongoDB. Thus, you should use MQL (and not SQL).

Please see an example here: https://docs.mindsdb.com/nlp/sentiment-analysis-inside-mongodb-with-openai

**Comment by martyna-mindsdb** [comment]
@akshay-bharadva 
Please see the review comment above and modify this tutorial to use MQL (instead of SQL).

File: docs/tutorials/sentiment-analysis-of-movie-reviews.mdx (hunk 1 above)
Comment (akshay-bharadva) [reply docs/tutorials/sentiment-analysis-of-movie-reviews.mdx:28]: @martyna-mindsdb , Sorry for using SQL, I've committed with MQL.

**Comment by martyna-mindsdb** [comment]
@akshay-bharadva 
Please see Zoran's comment above and implement the changes.

File: docs/tutorials/sentiment-analysis-of-movie-reviews.mdx [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+```
+
+## Training a Predictor
+
+We use the `openai` engine to predict the sentiment of the review.
+
+we can create a model for text sentiment analysis like this:
+
+```bash
+mongo_demo_db> db.models.insertOne({
```
Comment (martyna-mindsdb) [inline docs/tutorials/sentiment-analysis-of-movie-reviews.mdx:90]: This should be executed from mindsdb:

`mindsdb> db.models.insertOne({`

Please see this example: https://docs.mindsdb.com/nlp/sentiment-analysis-inside-mongodb-with-openai

Based on the example, please correct it throughout this article. The `insertOne`, `db.getCollection('models')`, etc. should be executed from `mindsdb>` and not from `mongo_demo_db>`.

File: docs/tutorials/sentiment-analysis-of-movie-reviews.mdx [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+```
+
+Here, we provided `prompt_template` option with mainly three sentiments, and those are `positive`, `negative` and `neutral`.
+
+## Status of a Predictor
+
+The next step is to check the status of a predictor. Once the above query has started execution, we can check the status of the creation process with the following query:
+
+```bash
+mongo_demo_db> db.getCollection('models').find({
```
Comment (martyna-mindsdb) [inline docs/tutorials/sentiment-analysis-of-movie-reviews.mdx:116]: As mentioned in the previous comment, it should be executed from `mindsdb>`.

File: docs/tutorials/sentiment-analysis-of-movie-reviews.mdx [hunk 4]
```diff
@@ -0,0 +1,200 @@
...
+
+## Making Predictions
+
+### Making a Single Prediction
+
+Once the creation is complete, the behavior is the same as with any other AI table.
+Here we are querying by specifying synthetic data in the actual query and the model we created i.e, `sentiment_classifier_openai`:
+
+```bash
+mongo_demo_db> db.sentiment_classifier_openai.find({
```
Comment (martyna-mindsdb) [inline docs/tutorials/sentiment-analysis-of-movie-reviews.mdx:131]: As mentioned in the previous comment, it should be executed from `mindsdb>`.

File: docs/tutorials/sentiment-analysis-of-movie-reviews.mdx [hunk 5]
```diff
@@ -0,0 +1,200 @@
...
+  sentiment: "neutral"
+}
+```
+
+### Making Batch Predictions
+
+Similarly, we can join model with another table for batch predictions with the following query:
+
+```bash
+mongo_demo_db> db.sentiment_classifier_openai.find(
```
Comment (martyna-mindsdb) [inline docs/tutorials/sentiment-analysis-of-movie-reviews.mdx:150]: As mentioned in the previous comment, it should be executed from `mindsdb>`.

**Comment by martyna-mindsdb** [comment]
@akshay-bharadva 
Please implement the changes mentioned in the review comments above.

**Pull request merged**